### PR TITLE
[FR-LiveTest] Update selection mark folder name 

### PR DESCRIPTION
**Pull request by mssfang**
- update the folder name to `formselectionmark`

### CODE REVIEW

File: sdk/formrecognizer/test-resources.json [hunk 1]
```diff
@@ -73,7 +73,7 @@
         },
         "selectionMarkTrainingDataContainer": {
             "type": "string",
-            "defaultValue": "selectionmark"
+            "defaultValue": "formselectionmark"
```
Comment (samvaity) [inline sdk/formrecognizer/test-resources.json:76]: Was this changed or we had the incorrect one before?

**Review by samvaity — APPROVED** [review]

File: sdk/formrecognizer/test-resources.json (hunk 1 above)
Comment (mssfang) [reply sdk/formrecognizer/test-resources.json:76]: the folder name changed

**Pull request merged**